Automattic/wp-e2e-tests
# Login: Add test for push 2fa case

Add a case for a user logging in with a push 2FA enabled.

This test case imitates the Android [app](https://play.google.com/store/apps/details?id=org.wordpress.android) getting a push with a token via [GCM](https://developers.google.com/cloud-messaging/) and then "approving" it via
WPCOM rest API. It then asserts the user is indeed logged in.

#### Testing instructions
  
1. Run `git checkout add/login-2fa-push`
2 Create/Modify `config/local-test.json`  to add the new accounts information (see pb/1d081). You can also use `local-development.json`.
- Run it for 1 device: `NODE_ENV=test BROWSERSIZE=mobile ./node_modules/.bin/mocha specs/wp-log-in-out-spec.js`

#### Reviews
